bc_obps/service/operation_service.py

```diff
@@ -100,6 +102,13 @@ def update_status(cls, user_guid: UUID, operation_id: UUID, status: Operation.St
             fields_to_update.append('submission_date')
         operation.status = Operation.Statuses(status)
         operation.save(update_fields=fields_to_update)
+        if operation.status == Operation.Statuses.REGISTERED:
+            send_registration_and_boro_id_email(
+                BoroEmailTemplateNames.CONFIRMATION,
+                operation.operator.legal_name,
+                operation,
+                UserDataAccessService.get_by_guid(user_guid),
+            )
```

Comment:
Nice 🤩 I'd recommend adding a comment here to explain why this change is being made.